site/docs/workflows/dart_package.md

```diff
@@ -97,6 +97,12 @@ The Dart package workflow consists of the following steps:
 
 **Default** `"vm"`
 
+## Secrets
+
+### `ssh_key`
+
+**Optional** A ssh key used to access private repositories when installing dependencies.
```

Comment:
I believe it is "an SSH", or at least that is how the Webfactory's documentation has it, see [here](https://github.com/webfactory/ssh-agent#why).

```suggestion
**Optional** An SSH key used to access private repositories when installing dependencies.
```